### PR TITLE
Add support for theme color modes

### LINKED ISSUE #18246: Feature: add a toggler element to switch between color schemes
### Is your feature request related to a problem? Please describe.

The feature request is related to #16969 (How can I change scheme for Metro theme?) and #16868 (Add a select field for Metro color schemes).

Bootstrap v5.3.0-alpha now supports color modes or themes.
I added new color schemes in my BooDark theme, like in Metro.
There is currently no way to change the color schemes in the GUI.

### Describe the solution you'd like

To allow users to switch between color schemes, a toggle element will need to be created.

### Describe alternatives you've considered

Right now, only solution is that described [in this comment](https://github.com/phpmyadmin/phpmyadmin/issues/16969#issuecomment-864187054)

### Additional context

- phpMyAdmin version:  5.2.1
- BooDark versions: 1.1.0 (latest release)

### PR DESCRIPTION
**Pull request by MauricioFauth**
- Fixes https://github.com/phpmyadmin/phpmyadmin/issues/18246

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
💯

File: public/themes/bootstrap/theme.json [hunk 1]
```diff
@@ -4,5 +4,6 @@
     "description": "Bootstrap theme for phpMyAdmin",
     "author": "phpMyAdmin developers",
     "url": "https://www.phpmyadmin.net/",
-    "supports": ["6.0"]
+    "supports": ["6.0"],
+    "colorModes": ["light", "dark"]
```
[the remark this answers is not in the document]
Comment (ibennetch) [reply public/themes/bootstrap/theme.json:8]: I suspect we shouldn't translate the modes; that can get really messy with multiple languages especially with a theme like Metro (would you translate "Redmond?"). Seems to me it's easier and less messy to not translate these values at all, but I'm not certain of that.

[the remark this answers is not in the document]
Comment (williamdes) [reply public/themes/bootstrap/theme.json:8]: > would you translate "Redmond?"

Nope

>  Seems to me it's easier and less messy to not translate these values at all, but I'm not certain of that.

Well, I think people will come at us and ask for translations of light and dark.
But less probably `Redmond` or other stuff that's not really an English word from a non english user

That's a bit complicated to decide

[the remark this answers is not in the document]
Comment (MauricioFauth) [reply public/themes/bootstrap/theme.json]: I added translations for Light and Dark, which I think will be the more common ones. Other colors names will not have translations.

**Review by williamdes — APPROVED** [review]

**Pull request merged**